src/App.tsx

```diff
@@ -1,5 +1,15 @@
+import TodoList, { Todo } from "./components/TodoList/TodoList";
```

Comment:
That's awesome!

Ah tip about it, to identify easily what is type and what is component, you can use the import like this:


```suggestion
import TodoList from "./components/TodoList/TodoList";
import type { Todo } from "./components/TodoList/TodoList";
```

So both are importing the same file, but the first one is for component and the second one (the one that has `type`) is for types.